package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "start": "npm-run-all --parallel startBackendDev buildFrontendDev",
+    "startProd": "npm-run-all --parallel startBackendProd buildFrontendProd",
```

Comment:
We haven't had any luck getting this run in parallel thing to work on LogicaHealth.